lib/iris/cube.py

```diff
@@ -1697,7 +1697,7 @@ def data(self):
             (10, 20)
 
         """
-        if self._numpy_array is None:
```

Comment:
We've defined a method to make this check, so let's use it rather than repeating the same code.